actors/common/src/main/java/org/sunbird/learner/util/ContentSearchUtil.java

```diff
@@ -39,7 +39,10 @@
         JsonKey.AUTHORIZATION, JsonKey.BEARER + System.getenv(JsonKey.SUNBIRD_AUTHORIZATION));
   }
 
-  public static Map<String, Object> searchContent(String body) throws Exception {
+  public static Map<String, Object> searchContent(String body, Map<String, String> headerMap)
+      throws Exception {
+    // adding all incoming header with predefine static header map.
+    headers.putAll(headerMap);
```

Comment:
Add fixed headers to incoming request headers map and use the same in the request.